src/gwt/src/org/rstudio/studio/client/workbench/views/console/ConsoleInterpreterVersion.java

```diff
@@ -167,24 +174,23 @@ else if (StringUtil.equals(type, ReticulateEvent.TYPE_REPL_TEARDOWN))
          adaptToR();
       }
    }
-   
-   
-   private String rVersionLabel()
+
+
+   private void setRVersionLabel()
    {
-      String version = constants_.unknownLabel();
-      
-      try
+      server_.getRVersion(new ServerRequestCallback<RVersionSpec>()
       {
-         version = session_
-               .getSessionInfo()
-               .getRVersionsInfo()
-               .getRVersion();
-      }
-      catch (Exception e)
-      {
-      }
-      
-      return "R " + version;
+         @Override
+         public void onResponseReceived(RVersionSpec versionSpec)
+         {
+            label_.setText("R " + versionSpec.getVersion());
+         }
+         @Override
+         public void onError(ServerError error)
+         {
+            Debug.logError(error);
```

Comment:
Although unlikely to actually get an error here, maybe should set something in the label if it does? Otherwise could end up stuck showing an out-of-date label and we'd not know it was happening. Thinking leave the logError and also add `label_.setText("Error fetching R version") or something.